src/styles/blocks/card.scss

```diff
@@ -0,0 +1,88 @@
+* {
+  font-family: "Roboto", "Helvetica", sans-serif;
+  font-size: 12px;
+  line-height: 1.5;
```

Comment:
you can move those to body

Reply:
Yes, I can. But what for? I also can move these properties to html, there more than one correct versions of code. Moving these properties to another selector will make my code longer. 
Mentor TarasHoliuk left another comment below this part of my code: "it should be applied to each element on the page (*)"

Reply:
That's why you can't use `*` before anything except `box-sizing: border-box'

Wide range of applications: The `*` selector selects all elements on the page. This means that any styles applied to the * selector will be applied to all elements, including inline elements such as `<script>`, `<style>`, and others. This can lead to unexpected problems and affect the behaviour of scripts and styles.

Impact on performance: Selecting all the elements on a page can lead to an increase in the amount of styles passed to the browser, which can lead to a decrease in site performance. The `*` selector can make styling slower, especially if you apply complex styles or increase the number of elements to be processed.

Overwrite styles: The `*` selector has the highest level of specificity, which means that it overrides (overwrites) other styles, including styles defined with classes or identifiers. This can lead to style conflicts and make it difficult to manage styles on a page.

Reply:
I don`t agree with the last paragraph. The * selector doesn`t impact the specificity weight value.
https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity